cns/service/main.go

```diff
@@ -805,6 +809,33 @@ func main() {
 		}
 	}
 
+	if cnsconfig.EnableAsyncPodDelete {
+		// Start fs watcher here
+		logger.Printf("[Azure CNS] Start fsnotify watcher for intended deletes")
+		cnsclient, err := cnsclient.New("", cnsReqTimeout) //nolint
+		if err != nil {
+			logger.Errorf("Failed to initialize CNS client:%v.\n", err)
+		}
+		w := &fsnotify.Watcher{
+			CnsClient: cnsclient,
+		}
+		watcherPath := cnsconfig.AsyncPodDeletePath
+
+		go func() {
+			for {
+				if err := fsnotify.WatchFs(w, watcherPath, z); err != nil {
+					logger.Errorf("Failed to start fsnotify watcher, err:%v.\n", err)
+				} else {
+					logger.Printf("Fsnotify watcher started")
```

Comment:
```suggestion
					logger.Print("Fsnotify watcher started")
```

Reply:
will keep this one unchanged since logger.Print is undefined